src/pages/example.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { Helmet } from 'react-helmet';
+import { CircleMarker, Rectangle, Marker, Popup } from 'react-leaflet';
+
+import Layout from 'components/Layout';
+import Container from 'components/Container';
+import Map from 'components/Map';
+import Snippet from 'components/Snippet';
+
+const json = [
+  {
+    coords: [51.7527, 0.3394],
+    place: 'St. Albans'
+  },
+  {
+    coords: [51.7356, 0.4685],
+    place: 'Chelmsford'
+  },
+  {
+    place: 'Canterbury',
+    coords: [51.2802, 1.0789]
+  }
+];
+
+const circleLocation = {
+  lat: 51.49,
+  lng: -0.08,
+};
+
+const rectangleLocation = [
+  [51.49, -0.08],
+  [51.5, -0.06],
+];
+
+const CENTER = [circleLocation.lat, circleLocation.lng];
+const DEFAULT_ZOOM = 10;
+
+const ExamplePage = () => {
+
+  async function mapEffect() {
```

Comment:
I think we can remove this function and from `mapSettings` since we're not actually using it